lib/exp.py

```diff
@@ -68,12 +100,18 @@ def list_experiments(self):
         return [Experiment.from_body(item) for item in response['items']]
```

Comment:
why aren't we also retrying things like line 95 above:
i.e. `self.k8s.list_namespaced_custom_object()` call?  Same goes for create/delete 
(sorry I can't highlight it directly since the section is collapsed).